### DISCUSSION TITLE
Bring chat window to the top on new messages

### DISCUSSION THREAD

**Discussion by bilogic** [rustdesk/rustdesk#7507] [General]
Version 1.2.4
Build date: 2024-02-28 00:36

During a remote session, it would help if new chat messages will cause the other side's chat window to surface to the top, for both controller and controlled.

Otherwise the remote person seeking help may sometimes be lost and unaware of new "tips" that I'm sending over on how to perform the action they are attempting.

**Comment by rustdesk** [comment]
I just tested nightly, both sides can show the chat window on top when receiving message.

**Reply by bilogic** [reply]
1.2.4? 2024-02-28?
